docs/releases/5.1.1.txt

```diff
@@ -18,3 +18,7 @@ Bugfixes
   children). A new :class:`~django.contrib.auth.forms.AdminUserCreationForm`
   including this field was added, isolating the feature to the admin where it
   was intended (:ticket:`35678`).
+
+* Restored the ability to override the undocumented methods
+  ``DatabaseWrapper.ensure_timezone`` and ``DatabaseWrapper.ensure_role``
+  (:ticket:`35688`).
```

Comment:
```suggestion
* Restored, following a regression in Django 5.1, the ability to override the
  undocumented methods ``DatabaseWrapper.ensure_timezone()`` and
  ``ensure_role()`` (:ticket:`35688`).
```

Reply:
Thank you! I also pushed an update to try and make this postgres specific